x/mint/types/params.go

```diff
@@ -171,8 +171,8 @@ func validateGoalBonded(i interface{}) error {
 		return fmt.Errorf("invalid parameter type: %T", i)
 	}
 
-	if v.IsNegative() {
-		return fmt.Errorf("goal bonded cannot be negative: %s", v)
+	if v.IsNegative() || v.IsZero() {
```

Comment:
Is this state compatible?

Reply:
I think ur right on these not being state breaking, closing this issue